packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/ModalDialog/index.js

```diff
@@ -39,6 +39,7 @@ const ModalDialog = ({ onToggle, data: { date, user, action } }) => {
           paddingLeft={6}
           paddingRight={6}
           background="neutral100"
+          borderRadius={pxToRem(4)}
```

Comment:
Sorry I was too slow 🤦🏼 I think you can simply use `hasRadius` here, since `Grid` extends `Box`: https://github.com/strapi/design-system/blob/b30d2bdee9b8be2eae723e940c2b97762b0fc7b5/packages/strapi-design-system/src/Box/Box.js#L39

Reply:
Noted 👍  Just updated in another PR here, https://github.com/strapi/strapi/pull/15170